packages/reports/build/webpack.config.js

```diff
@@ -0,0 +1,58 @@
+const path = require('path');
+const { ALIASES, IS_RELEASE, MINIMIZERS, plugins, rules } = require('./constants');
+
+module.exports = function (env) {
+    const base = env && env.base && env.base !== true ? `/${env.base}/` : '/';
+
+    return {
+        context: path.resolve(__dirname, '../'),
+        devtool: IS_RELEASE ? undefined : 'eval-cheap-module-source-map',
+        entry: {
+            reports: path.resolve(__dirname, '../src', 'index.jsx'),
+        },
+        mode: IS_RELEASE ? 'production' : 'development',
+        module: {
+            rules: rules(),
+        },
+        resolve: {
+            alias: ALIASES,
+            extensions: ['.js', '.jsx', '.ts', '.tsx'],
+        },
+        optimization: {
+            chunkIds: 'named',
+            moduleIds: 'named',
+            minimize: IS_RELEASE,
+            minimizer: MINIMIZERS,
+        },
+        output: {
+            filename: 'reports/js/[name].js',
+            publicPath: base,
+            path: path.resolve(__dirname, '../dist'),
+            chunkFilename: 'reports/js/reports.[name].[contenthash].js',
+            libraryExport: 'default',
+            library: '@deriv/reports',
+            libraryTarget: 'umd',
+        },
+        externals: [
+            {
+                react: 'react',
+                'react-dom': 'react-dom',
+                'react-router-dom': 'react-router-dom',
+                'react-router': 'react-router',
+                mobx: 'mobx',
+                'mobx-react': 'mobx-react',
+                '@deriv/shared': '@deriv/shared',
+                '@deriv/components': '@deriv/components',
+                '@deriv/translations': '@deriv/translations',
+                '@deriv/deriv-charts': '@deriv/deriv-charts',
+                '@deriv/account': '@deriv/account',
```

Comment:
Do we have any dependancies to Accounts package that we need this alias for?

Reply:
No, will remove it :pray: